docs/api_guide/plot_06_sklearn_pipeline_cv_demo.py

```diff
@@ -295,6 +350,9 @@ def highlight_max_cell(cvdf_wide, ax):
 sns.despine(ax=ax)
 
 # %%
+# !!! success
```

Comment:
while I like the "success" here, it's not consistent with how we previously interpreted the firing rate plots. I tend to think of these boxes as "extra info", rather than "core info", so I think it  should probably be removed. maybe add emojis instead? :tada: :tada: